Workbooks/UsageMetrics/Usage - Events/settings.json

```diff
@@ -0,0 +1,12 @@
+{
+    "$schema": "https://github.com/Microsoft/Application-Insights-Workbooks/blob/master/schema/settings.json",
+    "name": "Events",
+    "author": "Microsoft",
+    "galleries": [
+        {
+            "type": "usageMetrics",
```

Comment:
you're adding a new workbook type?  until someone also makes changes to enable this in the insightsportal extension, this should/would be showing you a warning in the portal that it is being ignored and using "workbook" instead.

Reply:
Yes, I will add a new workbook type.